components/fieldset/src/atoms/legend/index.js

```diff
@@ -22,7 +26,7 @@ const StyledLegend = styled('legend')(
   ({ size }) => {
     const actualSize = Number.isNaN(Number(size)) ? HEADING_SIZES[size] : size;
 
-    if ((size !== undefined) && !actualSize) {
+    if (size !== undefined && !actualSize) {
```

Comment:
this is less readable/comprehensible to me

i like use of brackets to make logic explicit.  relying on knowledge of operator precedence makes this stuff harder work

Reply:
this too is the same issue as with brackets to make mathematical operations clearer

unfortunately prettier just won't leave this stuff alone 😢